Analysing ../../SCRUBD/SCRUBD-CD/data/solidity_codes/0x0ceae0bd68f89e41a1ed5dc20ca930c0c41856d7.sol:EscrowGoods...

--- STDERR ---
PHI instruction need arguments but 0 was given

Traceback (most recent call last):
  File "/home/sh1kaku/lihao/conkas/conkas.py", line 108, in main
    traces = sym_exec.execute()
  File "/home/sh1kaku/lihao/conkas/sym_exec/symbolic_executor.py", line 51, in execute
    new_traces = self.__sym_exec_traces(traces_to_execute)
  File "/home/sh1kaku/lihao/conkas/sym_exec/symbolic_executor.py", line 64, in __sym_exec_traces
    new_blocks = self.__sym_exec_block(block_to_analyse, trace.state)
  File "/home/sh1kaku/lihao/conkas/sym_exec/symbolic_executor.py", line 93, in __sym_exec_block
    new_blocks = self.__sym_exec_instruction(instruction, state)
  File "/home/sh1kaku/lihao/conkas/sym_exec/symbolic_executor.py", line 116, in __sym_exec_instruction
    return func(instruction, state)
  File "/home/sh1kaku/lihao/conkas/sym_exec/instructions/rattle_instructions.py", line 58, in inst_phi
    raise Exception
Exception
